api/handlers_metrics.go

```diff
@@ -204,6 +207,36 @@ func parseQuery(r *http.Request, request cloudwatch.MetricsRequest) error {
 	}
 	request["end"] = end
 
+	height := int64(400)
+	if h, ok := queries["height"]; ok {
+		hint, err := strconv.ParseInt(h[0], 10, 64)
```

Comment:
Do we also need to check that this is > 0 (or possibly some other minimum height)?